MySQL.md

```diff
@@ -191,3 +197,36 @@ Here's typical wal-g configuration for that case:
 Restore procedure is straightforward:
 * start mysql (it's recommended to create new mysql instance)
 * fetch and apply desired backup using `wal-g backup-fetch "backup_name"`
+
+
+### MariaDB - using with `mariabackup`
+
+It's recommended to use wal-g with mariabackup tool in case of MariaDB for creating lock-less backups.
+Here's typical wal-g configuration for that case:
+```
+ WALG_MYSQL_DATASOURCE_NAME=user:pass@tcp(localhost:3305)/mysql                                                                                                                                      
+ WALG_STREAM_CREATE_COMMAND="mariabackup --backup --stream=xbstream --datadir=/var/lib/mysql"                                                                                                                               
+ WALG_STREAM_RESTORE_COMMAND="xbstream -x -C /var/backups/mariadb"                                                                                                                       
+ WALG_MYSQL_BACKUP_PREPARE_COMMAND="mariabackup --prepare --target-dir=/var/backups/mariadb"                                                                                              
+ WALG_MYSQL_BINLOG_REPLAY_COMMAND='mysqlbinlog --stop-datetime="$WALG_MYSQL_BINLOG_END_TS" "$WALG_MYSQL_CURRENT_BINLOG" | mysql'
+```
+
+For the restore procedure you can follow the major part of [the offical docs for full backup and restore](https://mariadb.com/kb/en/full-backup-and-restore-with-mariabackup/):
+* stop mariadb
+* clean a datadir (typically `/var/lib/mysql`)
+* fetch and prepare desired backup using `wal-g backup-fetch "backup_name"`
+* restore with either [`--copy-back`](https://mariadb.com/kb/en/mariabackup-options/#-copy-back) or [`--move-back`](https://mariadb.com/kb/en/mariabackup-options/#-move-back) argument of `mariabackup`:
+```
+mariabackup --move-back --target-dir=/var/backups/mysql
```

Comment:
Is this step required ? 
Wal-g restores backup already to datadir (/var/lib/mysql).
All you need is prepare it (recover with redo logs) with --prepare

Reply:
I wasn't really sure about this bit, so I thought that the things which were in the official docs were correct. I'll change it.

Reply:
Docs looks fine now.